src/pages/LandingPages/CorporateCampaign/CCLandingPage.vue

```diff
@@ -6,7 +6,7 @@
 			<!-- TODO: Add promo code entry input, if no promo query params exist and  no promo is applied -->
 			<campaign-status
 				v-if="!hideStatusBar"
-				class="corporate-campaign-landing__status tw-sticky tw-top-8 md:tw-top-9 tw-z-2"
+				class="corporate-campaign-landing__status tw-sticky tw-z-2"
```

Comment:
Does this make the status bar cover the header now? Or is it naturally falling below the header properly?

Reply:
![image](https://user-images.githubusercontent.com/3431240/203657187-6f4d015d-84d6-4bcf-8de9-ea3d8cc6644a.png)
It was actually looking like this when the user wasn't logged in (see image)
And when the user was logged in, the banner looked fine.